### DISCUSSION TITLE
Free Vue.js 3 Tailwind CSS admin dashboard template

### DISCUSSION THREAD

**Discussion by vikdiesel** [tailwindlabs/tailwindcss#5178] [Show and tell]
[![Free Vue 3 Tailwind CSS admin dashboard](https://justboil.me/images/one-vue3-tailwind/repository-preview-hi-res.png)](https://justboil.github.io/admin-one-vue-tailwind/)

Hi, just wanted to share **Admin One** — free Vue.js 3 Tailwind CSS admin dashboard template

* Built with **Vue.js 3**, **Tailwind CSS** framework & **Composition API** 
* SPA with **Vuex** & **Router**
* **Production CSS** is only **27kb**
* **Responsive** layout
* Reusable components
* Free under MIT License

#### GitHub

https://github.com/justboil/admin-one-vue-tailwind

#### Demo

https://justboil.github.io/admin-one-vue-tailwind/

**Comment by Edna110** [comment]
I recommend you a wonderful site reports that will help you write any written work  >>  [My Balance Now](https://www.mybalancenow.kim/)
